signer/src/storage/model.rs

```diff
@@ -621,6 +656,19 @@ pub enum TransactionType {
     Donation,
 }
 
+/// The types of transactions the signer is interested in.
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, PartialOrd, Ord, sqlx::Type, strum::Display)]
+#[sqlx(type_name = "txo_type", rename_all = "snake_case")]
+#[derive(serde::Serialize, serde::Deserialize)]
+#[strum(serialize_all = "snake_case")]
+#[cfg_attr(feature = "testing", derive(fake::Dummy))]
+pub enum TxoType {
+    /// An sBTC transaction on Bitcoin.
+    Signers,
```

Comment:
I personally would have kept this singular, feels weird every time I read it :sweat_smile:

Reply:
Yeah, I always read it as `signers'`. Singular felt grammatically incorrect, but maybe I should change it to singular, I don't feel strongly about it. I just want it to be consistent throughout the codebase.

Reply:
I don't feel strongly enough to block a PR for it, so I'll resolve this.

My general approach to naming of most things is to use the [uninflected](https://en.wikipedia.org/wiki/Uninflected_word) noun, so that's largely where my comment stems from :)